### PR TITLE
Small changes to test API

### PR DESCRIPTION
**Pull request by everamal**

### CODE REVIEW

File: src/app/components/register/register.component.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ export class RegisterComponent implements OnInit {
   SubmitData() {
     this.model.name = this.profileForm.get("fullName").value;
     this.model.image = this.profileForm.get("photoURL").value;
+    console.log(this.model);
```
Comment (alfavenegas) [inline src/app/components/register/register.component.ts]: No console logs in pull requests

Comment (everamal) [reply src/app/components/register/register.component.ts]: es provisional para comprobar que se envia el objeto

Comment (everamal) [reply src/app/components/register/register.component.ts]: o puedes ayudarme a poner un logger

Comment (Miguelsiqueiros) [inline src/app/components/register/register.component.ts]: We need too implement a logger here, we don't want our users to see this.

**Pull request merged**